src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1259,7 +1259,7 @@ CREATE TABLE dbo.ExportJob
     Status varchar(10) NOT NULL,
     HeartbeatDateTime datetimeoffset(7) NULL,
     QueuedDateTime datetimeoffset(7) NOT NULL,
-    RawJobRecord varbinary(max) NOT NULL,
+    RawJobRecord varchar(max) NOT NULL,
```

Comment:
I am currently storing the raw job record as a JSON string to facilitate updating the status in the `AcquireExportJobs` procedure. This *seems* to be alright since export jobs records are fairly small.

Reply:
I think that's fine.